codex/erasure/erasure.nim

```diff
@@ -63,10 +63,11 @@ type
     store*: BlockStore
 
 proc encode*(
-  self: Erasure,
-  manifest: Manifest,
-  blocks: int,
-  parity: int): Future[?!Manifest] {.async.} =
+    self: Erasure,
+    manifest: Manifest,
+    blocks: int,
+    parity: int
+): Future[?!Manifest] {.async.} =
```

Comment:
I definitely agree this is a much nicer way to pick out the return type. If we are going to make these formatting changes here, it would be my personally preference to do it uniformly across the entire codebase.

Reply:
Nice, yah I could remove them from this PR and do a bigger PR for that styling in the repo. I didn't want to undertake that as a first PR. :)

Reply:
Sounds like a good idea :)